### DISCUSSION TITLE
Just to see their record

### DISCUSSION THREAD

**Discussion by Akaki-1991** [YetiForceCompany/YetiForceCRM#16029] [How to]
Hello,
I wonder if we can place such a restriction that only the creator of the record can see and not his boss, because the role can be seen even in a closed module.

**Comment by s2b-git** [comment]
against logic I think. Boss is boss :-)

**Reply by Akaki-1991** [reply]
> against logic I think. Boss is boss :-)

yes my friend :D :D, 

module is private,only include personal info :(
